src/org/labkey/test/tests/SimpleModuleTest.java

```diff
@@ -467,6 +467,8 @@ private void doTestCustomFolder()
         assertTextPresentInThisOrder("A customized web part", "Data Pipeline", "Experiment Runs", "Sample Type", "Assay List");
         assertTextPresent("Run Groups");
         assertElementNotPresent(Locator.linkWithText("Create Run Group")); // Not in small Run Groups web-part.
+        // ensure the mouse isn't inadvertently hovering something else, causing obscuring popup
+        mouseOver(portalHelper.getBodyWebPart("A customized web part").getComponentElement());
```

Comment:
I would move the mouse off the web part, maybe to the icon at the top of the page.
```suggestion
        mouseOver(Locator.tagWithClass("a", "brand-logo"));
```